python/ray/serve/schema.py

```diff
@@ -186,6 +186,62 @@ def __eq__(self, other: Any) -> bool:
         return self._compute_hash() == other._compute_hash()
 
 
+@PublicAPI(stability="alpha")
+class TracingConfig(BaseModel):
+    """Tracing config for configuring serve tracing."""
+
+    class Config:
+        extra = Extra.forbid
+
+    export_path: str = Field(
```

Comment:
This field name is pretty confusing. I thought it was going to be a URI or file to which the traces should get written. Seems like it's more like `exporter_import_path`?